custom/icds_reports/reports/adolescent_girls.py

```diff
@@ -119,7 +122,7 @@ def get_adolescent_girls_sector_data(domain, config, loc_level, location_id, sho
     chart_data['blue'] = sorted(chart_data['blue'])
 
     return {
-        "tooltips_data": tooltips_data,
+        "tooltips_data": dict(tooltips_data),
```

Comment:
Why does this need a cast

Reply:
@emord because ```defaultdict``` with lambda function is not cachable by ```quickcache```.